packages/e2e-test-utils/src/switch-block-inspector-tab.js

```diff
@@ -9,11 +9,13 @@ export async function switchBlockInspectorTab( label ) {
 		`.block-editor-block-inspector__tabs button[aria-label="${ label }"]`
 	);
 
-	if ( tabButton ) {
-		const id = await page.evaluate( ( tab ) => tab.id, tabButton );
-		await tabButton.click();
-		await page.waitForSelector(
-			`div[role="tabpanel"][aria-labelledby="${ id }"]`
-		);
+	if ( ! tabButton ) {
+		throw new Error( `Missing block inspector tab: ${ label }.` );
 	}
+
+	const id = await page.evaluate( ( tab ) => tab.id, tabButton );
```

Comment:
Nit: Couldn't this just be:

```suggestion
	const id = await tabButton.getProperty( 'id' );
```

?

Reply:
👍 Thanks, I've updated this ( https://github.com/WordPress/gutenberg/pull/47097/commits/32bf77e37433212d13831b174511cfa5bed26c26 ), so it is a little easier to grok. The actual code that works is a little different than suggested but now also follows the patterns in other e2es.

```js
const id = await ( await tabButton.getProperty( 'id' ) ).jsonValue();
```